pyproject.toml

```diff
@@ -0,0 +1,93 @@
+[project]
+name = "wayback"
+# version = "0.0.1"
+dynamic=["version"]
+description = "Python API to Internet Archive Wayback Machine"
+# FIXME: validate this is the correct replacement for long_description and
+# long_description_content_type!
+# It seems this can also be:
+# readme = {content-type = "text/x-rst", file = "README.rst"}
+readme = "README.rst"
+maintainers = [
+  { name = "Rob Brackett", email = "rob@robbrackett.com" },
+  { name="Environmental Data Governance Initiative", email="EnviroDGI@protonmail.com" },
+]
+# TODO: put contributors in `authors` field:
+# authors = [
+#   { name="XYZ" },
+# ]
+# FIXME: how does this show up in PyPI? In setup.py we had:
+#   license="BSD (3-clause)"
+# But that's not the right format here. And there's no trove classifier for it.
+# Maybe this should be:
+#   license = {text = "BSD (3-clause)"}
+license = {file = "LICENSE"}
```

Comment:
Yeah, this should probably change. Here’s how the the file reference shows up on test.pypi.org:

<img width="278" alt="Screenshot from PyPI showing several lines of the full license text" src="https://github.com/edgi-govdata-archiving/wayback/assets/74178/479a1221-b2bf-41d4-b7e9-0449f63d296b">

vs. what we currently have with `setup.py`:

<img width="257" alt="Screenshot from PyPI showing just the name of the license" src="https://github.com/edgi-govdata-archiving/wayback/assets/74178/503c3038-fc82-4924-87b6-1c05437304d5">

Reply:
Changed this and uploaded to test.pypi.org; it fixes the issue!